back/engines/free/email_campaigns/app/serializers/email_campaigns/web_api/v1/campaign_serializer.rb

```diff
@@ -70,6 +70,12 @@ class WebApi::V1::CampaignSerializer < ::WebApi::V1::BaseSerializer
       end
     end
 
+    attribute :stats, if: proc { |object|
+      object.manual? && object.sent?
```

Comment:
Just wondering, what would be the reason(s) to not include this for automated campaigns?

Reply:
The FE doesn't use it.